src/android/java/io/jxcore/node/BtToServerSocket.java

```diff
@@ -13,33 +13,39 @@
  */
 public class BtToServerSocket extends Thread implements StreamCopyingThread.CopyThreadCallback {
```

Comment:
For catch- lets chat, and I'll fix things if needed really quickly after the chat. For mHTTPPort, in this case we are actually getting port externally set, so we need to use variable in here.